metrics/exporter.go

```diff
@@ -85,6 +85,15 @@ func UpdateExporterFromConfigMap(component string, logger *zap.SugaredLogger) fu
 	return ConfigMapWatcher(component, nil, logger)
 }
 
+// RegisterResourceView is placeholder for real implementation in https://github.com/knative/pkg/pull/1392.
+// That PR will introduce a breaking change, as we need to  all view.Register to RegisterResourceView in
```

Comment:
> we need to  all view.Register to RegisterResourceView

Something is missing from that sentence

Reply:
```suggestion
// That PR will introduce a breaking change, as we need to convert some view.Register to RegisterResourceView in
```

We don't need to change all calls, only those that report on behalf of additional resources. (i.e. GC stats don't need to be changed, but per-Trigger, per-Broker, or per-Revision views will need to be updated.)